tests/torch/runall.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Python virtual environment
+PYTHON_VENV="../../.venv-pytorch"
+
+# Clear SLURM logs (*.out and *.err files)
+rm -rf logs_slurm
+mkdir logs_slurm
+rm -rf logs_torchrun logs_mpirun logs_srun
+
+export MNIST_PATH="/ceph/hpc/data/st2301-itwin-users/mbunino/mnist" #"/p/project1/intertwin/smalldata/mnist"
```

Comment:
I guess these file paths are meant to be specific to mbunino because of the way the Vega file structure is set up, right?

Reply:
You can change that path when you are using the script. Anyways, this script is for development only. I have added a disclaimer to the top